tests/behat/features/userforms.feature

```diff
@@ -66,6 +68,7 @@ Feature: Userforms
     # Drag and drop my text field 2 to Page Two
     Then I drag the ".ss-gridfield-item[data-id='4'] .handle" element to the ".ss-gridfield-item[data-id='6'] .handle" element
     And I press the "Publish" button
+    And I dismiss all toasts
```

Comment:
This lets the behat test run correctly in kitchen sink - otherwise the toasts block the tab which we click in the next step.
Requires https://github.com/silverstripe/silverstripe-framework/pull/10661